doc/lsst.meas.base/intro.rst

```diff
@@ -0,0 +1,94 @@
+##################################
+Overview of the measurement system
+##################################
+
+The meas_base package provides the basis of LSST's system for performing measurements on astronomical images.
+This document provides a brief overview of how the pieces of the meas_base framework fit together.
+It is intended to help new users orient themselves rather than to provide a comprehensive user guide.
+
+The structure of meas_base measurement
+======================================
+
+Measurements take place within the meas_base framework.
+
+meas_base distinguishes between a measurement “algorithm” (i.e. a bit of code which takes some pixels and returns some useful result based upon them) and a measurement “driver”, which takes an image and runs a bunch of algorithms on it.
+
+Typical measurement algorithms include (say) PSF fluxes, the SDSS shape algorithm, etc.
+
+The measurement driver depends on the type of measurement being performed.
+meas_base provides two: single frame measurement, which is measurement on a single visit, and forced measurement, which takes an image and an external catalog which it uses to fix source centroids and shapes.
+
+Measurement drivers are implemented as tasks, while algorithms are meas_base plugins.
+In most (but not all) cases, algorithms and drivers can be compiled fairly arbitrarily; that is, the PSF flux algorithm can be used with single frame measurement or forced measurement, etc.
+
+meas_base provides those two drivers and a bunch of simple algorithms.
+Other packages can provide more of either, following the same conventions.
+Most of our meas_extensions packages provide measurement algorithms for use in the meas_base framework (e.g. simpleShape, shapeHSM, etc).
+
+A notable extension package is ip_diffim.
+This provides a driver suitable for meas_base-style measurement on difference images and a bunch of algorithms (e.g. dipole measurement) which are of relevance in that context.
+
+Naming and selecting measurement algorithms
+===========================================
+
+The new users can generally skip over the details of measurement drivers: we have the ones we need for now, and they mostly “just work”.
+The interesting stuff all happens in algorithms.
+
+Understand first that algorithms may be referred to by strings which describe both their home package and their name.
+Thus, ``base_PsfFlux`` refers to the ``PsfFlux`` algorithm in :py:mod:`lsst.meas.base`; ``ip_diffim_DipoleFit`` refers to the ``DipoleFit`` algorithm in :py:mod:`lsst.ip.diffim`.
+
+When configuring a measurement driver (i.e. a task like :py:class:`lsst.meas.base.sfm.SingleFrameMeasurementTask`) we specify a list of plugins to be run when executing the driver.
```

Comment:
I'm not sure I agree with all of the stylistic choices above, but I'm happy enough to go along with them. However, since `plugins` is a code literal and it's awkawrd to combine that with a link in reST, I'm  reluctant to do so. I've reworded not to use “here”, but I don't think it's really an improvement.